src/components/ArticleGrid.jsx

```diff
@@ -28,10 +40,32 @@ export default function ArticleGrid({ topic }) {
       return newArticles;
     });
   };
+  const toggleOrder = () => {
+    setOrder((prevOrder) => (prevOrder === "asc" ? "desc" : "asc"));
+  };
+  const handleSortChange = (event) => {
+    setSortBy(event.target.value);
+  };
   if (isLoading) return <p>Loading...</p>;
   return (
     <>
       <section>
+        <Toolbar sx={{ padding: "1em" }}>
+          <Stack direction="row" spacing={2} sx={{ marginLeft: "auto" }}>
+            <Stack direction="row" alignItems="center" spacing={2}>
+              <Typography variant="h6">Sort by:</Typography>
+              <Select value={sortBy} onChange={handleSortChange}>
+                <MenuItem value="created_at">Date</MenuItem>
```

Comment:
These menu items need a different colour styling as, presently the selection can't be seen.